cirq/google/ops/physical_z_tag.py

```diff
@@ -42,3 +42,6 @@ def _json_dict_(self) -> Dict[str, Any]:
 
     def __eq__(self, other) -> bool:
         return isinstance(other, PhysicalZTag)
+
+    def __hash__(self) -> int:
+        return 123
```

Comment:
Why `123`?

Reply:
Just an arbitrary value so that all instances hash to the same value.